frontend/src/pages/modelRegistry/screens/components/DeployRegisteredModelModal.tsx

```diff
@@ -68,12 +70,21 @@ const DeployRegisteredModelModal: React.FC<DeployRegisteredModelModalProps> = ({
         modelRegistryApi.api,
         modelVersion.id,
         modelVersion.registeredModelId,
+        rm?.customProperties || {},
```

Comment:
Also (and this applies to everywhere we'll pass in the objects to these `bump` utils) since they're not optional, we shouldn't fall back to a blank value like this. We can just narrow the type within the scope of calling these utils so we know they're defined, like by wrapping the call in a `if (rm)`. In this case, we can edit the early return a few lines above this (currently `if (!modelVersion.registeredModelId)`) so it returns early if `rm` isn't defined.